testcases/bestest_hydronic_heat_pump/models/find_days.py

```diff
@@ -0,0 +1,17 @@
+'''
+Created on Feb 5, 2021
+
+@author: Javier Arroyo
+
+Finds the peak and typical heat days for the bestest_hydronic_heatpump
+case. This case needs to be deployed.
+
+'''
+
+from data.find_days import find_days
+import json
+
+days = find_days(heat='reaPHeaPum_y', cool=None, data='simulate')
```

Comment:
I think we would rather want the thermal load to be used instead of electrical or gas power, so as not to take COP changes (if they exist) into account.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/285/commits/78056613cfdda5b15e0204c039d2fd98b6a23c92.